test/framework/kubernetesversions/data/debian_injection_script.envsubst.sh.tpl

```diff
@@ -88,7 +88,9 @@ if [[ "$${KUBERNETES_VERSION}" != "" ]]; then
     PACKAGE_VERSION="$(apt-cache madison kubelet | grep "$${VERSION_REGEX}-" | head -n1 | cut -d '|' -f 2 | tr -d '[:space:]')"
     for CI_PACKAGE in "$${PACKAGES_TO_TEST[@]}"; do
       echo "* installing package: $${CI_PACKAGE} $${PACKAGE_VERSION}"
+      apt-mark unhold "$${CI_PACKAGE}"
       apt-get install -y "$${CI_PACKAGE}=$${PACKAGE_VERSION}"
+      apt-mark hold "$${CI_PACKAGE}"
```

Comment:
We probably should only hold if it was held before we Unhold

Reply:
I would say it is safer to hold regardless. This would only be used when requesting a specific version to be installed, so holding at that version makes sense IMO.

Reply:
Good point

Reply:
Also a script only for CI so shouldn't matter